packages/neos-ui-guest-frame/src/initializePropertyDomNode.js

```diff
@@ -0,0 +1,61 @@
+import {$get} from 'plow-js';
+
+import {actions} from '@neos-project/neos-ui-redux-store';
+
+import {getGuestFrameWindow, closestContextPathInGuestFrame} from './dom';
+
+export default ({store, globalRegistry, nodeTypesRegistry, inlineEditorRegistry, nodes}) => propertyDomNode => {
+    const guestFrameWindow = getGuestFrameWindow();
+    if (!guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors']) {
+        guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors'] = {};
+    }
+
+    const initializedInlindeEditorApis = guestFrameWindow['@Neos.Neos.Ui:InitializedInlineEditors'];
+    const propertyName = propertyDomNode.getAttribute('data-__neos-property');
+    const contextPath = closestContextPathInGuestFrame(propertyDomNode);
+    const nodeTypeName = $get([contextPath, 'nodeType'], nodes);
+    const nodeType = nodeTypesRegistry.get(nodeTypeName);
+    const isInlineEditable = $get(['properties', propertyName, 'ui', 'inlineEditable'], nodeType) !== false;
+
+    if (isInlineEditable) {
+        const editorIdentifier = nodeTypesRegistry.getInlineEditorForProperty(nodeTypeName, propertyName);
+        const editorOptions = nodeTypesRegistry.getInlineEditorOptionsForProperty(nodeTypeName, propertyName);
+        const {bootstrap, createInlineEditor} = inlineEditorRegistry.get(editorIdentifier);
+
+        if (!initializedInlindeEditorApis[editorIdentifier] && bootstrap) {
+            try {
+                const {
+                    setFormattingUnderCursor,
+                    setCurrentlyEditedPropertyName
+                } = actions.UI.ContentCanvas;
+
+                bootstrap({
+                    setFormattingUnderCursor:
+                        (...args) => store.dispatch(setFormattingUnderCursor(...args)),
+                    setCurrentlyEditedPropertyName:
+                        (...args) => store.dispatch(setCurrentlyEditedPropertyName(...args))
+                });
+
+                initializedInlindeEditorApis[editorIdentifier] = true;
+            } catch (err) {
```

Comment:
The entire function is executed in a saga-context. Since we're fiddeling with the guest frame at that point, there might be plenty of exceptions completely unknown to us, that would become invisible or at least hard to read, if we do not display them explicitly like this.

I'll add an inline comment, to make that more transparent :)